Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:14.04
+
+ENV POSTGREST_VERSION 0.3.0.4
+ENV POSTGREST_SCHEMA public
+ENV POSTGREST_ANONYMOUS postgres
+ENV POSTGREST_JWT_SECRET thisisnotarealsecret
+ENV POSTGREST_MAX_ROWS 1000000
+ENV POSTGREST_POOL 200
+
+RUN apt-get update
+RUN apt-get install -y tar xz-utils wget libpq-dev
+
+RUN wget http://github.com/begriffs/postgrest/releases/download/v${POSTGREST_VERSION}/postgrest-${POSTGREST_VERSION}-ubuntu.tar.xz
+RUN tar --xz -xvf postgrest-${POSTGREST_VERSION}-ubuntu.tar.xz
+RUN mv postgrest /usr/local/bin/postgrest
```

Comment:
The wget, tar, mv sequence could be a one-liner like this
```sh
curl -L http://github.com/begriffs/postgrest/releases/download/v${POSTGREST_VERSION}/postgrest-${POSTGREST_VERSION}-ubuntu.tar.xz | tar zx -C /usr/local/bin/postgrest
```

But maybe there's a benefit to splitting commands into multiple RUN statements, not sure.